Gemfile

```diff
@@ -35,6 +35,7 @@ gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
 gem 'deacon', '~> 1.0'
 gem 'webpack-rails', '~> 0.9.8'
+gem 'mail', '~> 2.6.4'
```

Comment:
This is tighter than the Rails (actionmailer) dependency, can you relax it to ~> 2.6 perhaps? It'll cause problems with packaging where we use 2.6.1 in some cases.